ledger/catchpointtracker.go

```diff
@@ -159,6 +159,12 @@ func (ct *catchpointTracker) loadFromDisk(l ledgerForTracker, lastBalancesRound
 	ct.log = l.trackerLog()
 	ct.dbs = l.trackerDB()
 
+	ct.roundDigest = nil
+	ct.catchpointWriting = 0
+	// keep these channel closed if we're not generating catchpoint
+	ct.catchpointSlowWriting = make(chan struct{}, 1)
+	close(ct.catchpointSlowWriting)
```

Comment:
Hrmm ... maybe renaming the loadFromDisk tracker interface method to "reloadFromDisk" or "resetFromDisk" or something to make it clear for future authors that a tracker could have this method called multiple times during its lifetime?

Reply:
I'm not opposing to that. I think that the existing method description in the interface definition already define that.